src/Features/VisualBasic/Portable/Structure/Providers/MultiLineIfBlockStructureProvider.vb

```diff
@@ -1,22 +1,48 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+Imports System.Collections.Immutable
 Imports System.Threading
 Imports Microsoft.CodeAnalysis.Options
 Imports Microsoft.CodeAnalysis.PooledObjects
 Imports Microsoft.CodeAnalysis.Structure
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
-    Friend Class MultiLineIfBlockStructureProvider
-        Inherits AbstractSyntaxNodeStructureProvider(Of MultiLineIfBlockSyntax)
-
-        Protected Overrides Sub CollectBlockSpans(node As MultiLineIfBlockSyntax,
-                                                  spans As ArrayBuilder(Of BlockSpan),
-                                                  options As OptionSet,
-                                                  cancellationToken As CancellationToken)
-            spans.AddIfNotNull(CreateBlockSpanFromBlock(
-                               node, node.IfStatement, autoCollapse:=False,
-                               type:=BlockTypes.Conditional, isCollapsible:=True))
-        End Sub
+    Friend NotInheritable Class MultiLineIfBlockStructureProvider
+        Inherits InternalStructureBlockStructureProvider(Of
+            MultiLineIfBlockSyntax,
+            IfStatementSyntax,
+            ElseIfBlockSyntax,
+            ElseBlockSyntax,
+            ElseStatementSyntax,
+            EndBlockStatementSyntax)
+
+        Friend Overrides Function HeaderOfFullBlock(ifBlock As MultiLineIfBlockSyntax) As IfStatementSyntax
+            Return ifBlock.IfStatement
+        End Function
+
+        Friend Overrides Function GetInnerBlocks(ifBlock As MultiLineIfBlockSyntax) As SyntaxList(Of ElseIfBlockSyntax)
+            Return ifBlock.ElseIfBlocks
+        End Function
+
+        Friend Overrides Function InnerBlock_Text([ElseIf] As ElseIfBlockSyntax) As String
```

Comment:
rename `[ElseIf]` to elseIfBlock to avoid keyword escaping and to be consistent with `ifBlock`, which you used elsewhere.